### ISSUE TITLE
Leaderboard

### ISSUE THREAD

**Issue by AmrutaJayanti** [GSSoC24/Contributor#500]
@sanjay-kv I dont know why is my PR count going down and my PRs getting deleted. Is there any issue ? Sorry for the inconvenience if there are any spams by mistake as I am totally newbie to github and open-source . Can you please check is there any issue with my PRs . I have merged nearly 43 PRs but count came to 39 today

**Comment by sanjay-kv** [comment]
let me know the deleted PR repo,, i will check this

**Comment by AmrutaJayanti** [comment]
[Learn Python](https://github.com/animator/learn-python)

Some of the PRs from codeharbourhub 

I would like to mention [Link](https://github.com/animator/learn-python/blob/main/contrib/advanced-python/oops.md) , this is one of my documentation that has been merged. 

[Link](https://github.com/animator/learn-python/pull/395#issuecomment-2142458461)

**Comment by sanjay-kv** [comment]
![image](https://github.com/user-attachments/assets/82ad7eb2-387a-4371-91cf-ce779f54d466)

**Comment by AmrutaJayanti** [comment]
But @sanjay-kv my PRs are getting removed from leaderboard for those related to codeharborhub

![Screenshot 2024-07-25 174405](https://github.com/user-attachments/assets/4219a990-deec-4f09-a211-6e83765243b2)

**Comment by AmrutaJayanti** [comment]
I have merged many PRs from this ...

**Comment by sanjay-kv** [comment]
@Ajay-Dhangar can you look into this

**Comment by Ajay-Dhangar** [comment]
Yes, and I also work on whether contributors reopen PRs and create new PRs, and they mention PRs that were not counted in the leaderboard. When that happens, I reassign the contributor's levels based on their previous PR.

**Comment by AmrutaJayanti** [comment]
@sanjay-kv I think theres some issue with the Leaderboard , theres sudden reduce in points but showing all of my PRs . It even removed my badge . Is there really some issue with it?

**Comment by AmrutaJayanti** [comment]
@sanjay-kv Could you please help me with this ?
